### PR TITLE
Added disqualification status information

### PR DESCRIPTION
**Pull request by Cmdrd**
This will return one of two values:
```
Disqualified: 1
Not disqualified: 0
```
Returns metric as:
```
storj_sat_disqualified{instance="1.2.3.4",job="storj",satellite="118UWpMCHzs6CvSgWd9BfFVjw5K9pZbJjkfZJexMtSkmKxvvAW"} | 0
```

### CODE REVIEW

**Comment by Cmdrd** [comment]
Unfortunately (or fortunately I guess?) I do not have a disqualified node to test this against. This returns 0 for sure for non-disqualified nodes and anything else being reported should flag as disqualified. Not as cut and dry for specifying the values as I would like it to be but that is about all I can do given my current environment operation. If someone does have the value for a disqualified node then that can get merged in later.

File: storj-exporter.py [hunk 1]
```diff
@@ -112,6 +113,14 @@ def collect(self):
...
         storj_sat_day_storage.add_metric(["atRestTotal", sat], self.sat_data[sat]['storageDaily'][-1]['atRestTotal'])
     
+    for satellite in self.data['satellites']:
+      value = str(satellite['disqualified'])
+      if value == "None":
+        value = 0
+      else:
+        value = 1
+      storj_sat_disqualified.add_metric([satellite['id']], value)
+
```
Comment (anclrii) [inline storj-exporter.py]: See if you can use  `add_iterable_metrics` method in the existing for loop for satellites instead.

Also I don't think we need to be strict about the value returned. I prefer to keep exporter just translating values from api transparently where possible (unless it's breaking something).  In this case we can set anything that's not `0` to show as `disqualified` on Grafana side and let exporter just return api values. What do you think?

Comment (Cmdrd) [reply storj-exporter.py]: Ok, I will rework it to use that method. I tried yesterday but was having variable recasting problems as well as not having a good metric to present the string value. Thinking about it for a bit more though I could use a gauge metric with multiple labels and have a label for satellite and another for disqualified. The PromQL query would then be on the satellite returning the disqualified label as exactly what the API is returning.
That's actually super doable, I will redo this work and implement that. Thanks for the feedback.

**Comment by Cmdrd** [comment]
Closed PR. Will rework with suggestions.

**Pull request closed without merging**